yaml/ros2-turtlesim.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ros2-turtlesim-gui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ros2-turtlesim-gui
+  template:
+    metadata:
+      labels:
+        app: ros2-turtlesim-gui
+    spec:
+      containers:
+      - image: tomoyafujita/ros:rolling
+        command: ["/bin/bash", "-c"]
+        args: ["source /opt/ros/$ROS_DISTRO/setup.bash && ros2 run turtlesim turtlesim_node"]
+        imagePullPolicy: IfNotPresent
+        tty: true
+        name: ros2-turtlesim-gui
+        ### mounting host file to container
+        volumeMounts:
+        - mountPath: /tmp/.X11-unix
+          name: xserver
+        ### bind host environment variable to container
+        env:
+          - name: DISPLAY
+            value: ":1" # can this be host variable instead?
```

Comment:
this is minor concern, it would be probably better to get this value with asking host system during container initialization. but i am not sure if that is doable.